spec/support/test_server.rb

```diff
@@ -57,13 +57,21 @@ def echo_app_setup(response_code = 200)
       end
     end
 
+    def certificate_chain(domain)
+      ca = Billy.certificate_authority.cert
+      cert = Billy::Certificate.new(domain)
+      chain = Billy::CertificateChain.new(domain, cert.cert, ca)
+      [chain.file, cert.key_file]
```

Comment:
Let's just return a hash and use directly here: `http_server.ssl_options = certificate_chain(domain)`

Reply:
Sure.